test/property.test.js

```diff
@@ -45,7 +66,6 @@ describe('loopback:property generator', function() {
       customItemType: '', // temporary workaround for
                           // https://github.com/yeoman/generator/issues/600
       type: 'boolean',
-      required: 'true',
```

Comment:
That change is in fact quite related (to setting required attribute only when true, but leaving it out completely, otherwise in the generated/stored model), and even reverting would imply another apparently unrelated change to other test cases, which will now require to expect the mode to unnecessary contain `required: 'false'`...